src/rust/engine/process_execution/src/remote_tests.rs

```diff
@@ -2735,3 +2823,7 @@ fn assert_cancellation_requests(
     .collect::<Vec<_>>();
   assert_eq!(expected, cancels);
 }
+
+fn one_second() -> Option<Duration> {
+  Some(Duration::from_millis(1000))
+}
```

Comment:
More for curiosity. Why make this a function rather than constant?

Reply:
Would need to add a `lazy_static` dep, I think. Rust `static` fields are truly static in the binary, so they cannot have initialization code at runtime without indirection via `lazy_static`. More and more code is being marked [const](https://doc.rust-lang.org/std/keyword.const.html) over time, which means that it can be run at compile time to initialize the value... but I'm not sure `Duration::from_millis` has been yet.

Reply:
Ah, that makes sense. Thank you for the explanation!